script/demoScripts/demoGlacis.ts

```diff
@@ -0,0 +1,159 @@
+import { getContract, parseUnits, Narrow, zeroAddress, parseEther } from 'viem'
+import { randomBytes } from 'crypto'
+import dotenv from 'dotenv'
+import config from '../../config/glacis.json'
+import erc20Artifact from '../../out/ERC20/ERC20.sol/ERC20.json'
+import glacisFacetArtifact from '../../out/GlacisFacet.sol/GlacisFacet.json'
+import { GlacisFacet, ILiFi } from '../../typechain'
+import airliftArtifact from '../../out/IGlacisAirlift.sol/IGlacisAirlift.json'
+
+import { SupportedChain } from './utils/demoScriptChainConfig'
+import {
+  ensureBalance,
+  ensureAllowance,
+  executeTransaction,
+  setupEnvironment,
+  getConfigElement,
+  zeroPadAddressToBytes32,
+} from './utils/demoScriptHelpers'
+
+dotenv.config()
+
+// #region ABIs
+
+const ERC20_ABI = erc20Artifact.abi as Narrow<typeof erc20Artifact.abi>
+const GLACIS_FACET_ABI = glacisFacetArtifact.abi as Narrow<
+  typeof glacisFacetArtifact.abi
+>
+const AIRLIFT_ABI = airliftArtifact.abi as Narrow<typeof airliftArtifact.abi>
+
+// #endregion
+
+dotenv.config()
+
+async function main() {
+  // === Set up environment ===
+  const srcChain: SupportedChain = 'arbitrum'
+  const destinationChainId = 10
+
+  const {
+    client,
+    publicClient,
+    walletAccount,
+    lifiDiamondAddress,
+    lifiDiamondContract,
+  } = await setupEnvironment(srcChain, GLACIS_FACET_ABI)
+  const signerAddress = walletAccount.address
+
+  // === Contract addresses ===
+  const SRC_TOKEN_ADDRESS =
```

Comment:
Besides W, it might be worthwhile to test with additional tokens to ensure compatibility with transporters beyond Wormhole. For example, MIM (0xB153FB3d196A8eB25522705560ac152eeEc57901 on OP) or LINK (0x88fb150bdc53a65fe94dea0c9ba0a6daf8c6e196 on BASE).

Reply:
Sure! We have LINK tests as well - https://github.com/lifinance/contracts/blob/implement-glacis-LF-11761/test/solidity/Facets/GlacisFacet.t.sol#L376-L405